codewof/programming/content/en/driver-speed/solution.py

```diff
@@ -0,0 +1,7 @@
+def check_speed(speed):
+    if speed <= 50:
+        print('Ok.')
```

Comment:
Not much of an issue, but the other two strings have " "

Reply:
> ... and the entire concept of the `input` function with no hint about it - it took weeks for me to remember what the function was actually called
> 

I think I actually only remembered it because I'd seen it in other questions! I think it's a very common one for them (teachers and students) to use in NCEA and when they are learning though.

Reply:
No idea why this comment showed up here...